src/infrastructure/content/IContentCommunicator.java

```diff
@@ -0,0 +1,26 @@
+package infrastructure.content;
+
+public interface IContentCommunicator {
+	/*
+	 * UI will call this method with one json formatted string argument
+	 * @param : userDetails - It is json string which will contain server ip_address, username, image
+	 */
+	void initialiseUser (String userDetails);
+	
+	/*
+	 * UI will call this to paas the message of client to everyone else
+	 * @param : message - it is the actual message as a string
+	 */
+	void sendMessageToContent (String message);

```

Comment:
This is a JSON string too. In the future they can send more than strings

Reply:
for now, its like they will send message only.

Reply:
We need to think accomodating the future needs too